packages/roosterjs-content-model/lib/publicApi/utils/selectWord.ts

```diff
@@ -0,0 +1,103 @@
+import { ContentModelBlockGroup } from '../../publicTypes/group/ContentModelBlockGroup';
+import { ContentModelSegment } from '../../publicTypes/segment/ContentModelSegment';
+import { ContentModelText } from '../../publicTypes/segment/ContentModelText';
+import { createText } from '../../modelApi/creators/createText';
+
+export function selectWord(
+    path: ContentModelBlockGroup[],
+    marker: ContentModelSegment
+): ContentModelSegment[] {
+    const parent = path[0];
+    const block = parent.blocks[0];
+
+    if (block.blockType == 'Paragraph') {
+        const segments: ContentModelSegment[] = [];
+        let markerSelectionIndex = block.segments.indexOf(marker);
+        for (let i = markerSelectionIndex - 1; i >= 0; i--) {
+            const currentSegment = block.segments[i];
+            if (currentSegment.segmentType == 'Text') {
+                const found = findDelimiter(currentSegment, false);
+                if (found != null) {
+                    if (found == currentSegment.text.length) {
+                        break;
+                    }
+                    splitTextSegment(block.segments as ContentModelText[], i, found);
+                    //block.segments[i + 1].isSelected = true;
```

Comment:
remove this unneeded comment

Reply:
It is needed as I outlined in my previous comment. I would prefer if it was addressed before dismissing it.